go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -355,14 +370,14 @@ func (bnc *BaseNetworkController) allocateNodeSubnets(node *kapi.Node,
 
 // UpdateNodeAnnotationWithRetry update node's hostSubnet annotation (possibly for multiple networks) and the
 // other given node annotations
-func (bnc *BaseNetworkController) UpdateNodeAnnotationWithRetry(nodeName string, hostSubnetsMap map[string][]*net.IPNet,
+func (cnci *CommonNetworkControllerInfo) UpdateNodeAnnotationWithRetry(nodeName string, hostSubnetsMap map[string][]*net.IPNet,
```

Comment:
because dummy network controller does not have full information of BaseNetworkController, and this function is called by dummy network controller, and I want to make that clear.

Reply:
I see. I guess there are other candidate methods that could go into `CommonNetworkControllerInfo` which for me looks like they should be free utility methods instead of methods with a receiver. We can improve this later though.